lms/djangoapps/teams/static/teams/js/spec/topic_card_spec.js

```diff
@@ -0,0 +1,41 @@
+(function (define) {
+    'use script';
+
+    define(['jquery',
+            'underscore',
+            'teams/js/views/topic_card',
+            'teams/js/models/topic'],
+        function ($, _, TopicCardView, Topic) {
+
+            describe('topic card view', function () {
+                var view;
+
+                beforeEach(function () {
+                    view = new TopicCardView({
+                        model: new Topic({
+                            'id': 'renewables',
+                            'name': 'Renewable Energy',
+                            'description': 'Explore how changes in renewable energy will affect our lives.',
```

Comment:
It would be good to have more problematic characters, such as ``<`` and Unicode characters too.